app/common/alerts.py

```diff
@@ -14,12 +13,20 @@ class AlertViaGovNotify:
 
     @staticmethod
     def send(email, reference):
-        notifications_client = NotificationsAPIClient(settings.NOTIFICATION_COMBINED_KEY)
-        notifications_client.send_email_notification(email_address=email,
-                                                     template_id=settings.NOTIFICATION_TEMPLATE_ID,
-                                                     personalisation=None,
-                                                     reference=reference)
-        logger.info('Sent secure message email notification', user_uuid=g.user.user_uuid)
+
+        notification = {
+            "emailAddress": email,
+            "reference": reference
+        }
+
+        url = urlparse.urljoin(settings.RM_NOTIFY_GATEWAY_URL, settings.NOTIFICATION_TEMPLATE_ID)
+
+        auth = (settings.SECURITY_USER_NAME, settings.SECURITY_USER_PASSWORD)
+
+        response = requests.post(url, auth=auth, timeout=settings.REQUESTS_POST_TIMEOUT, json=notification)
+
+        log.info('Sent secure message email notification{} sent via RM Notify-Gateway to GOV.UK Notify.'
```

Comment:
I'm not sure how up to date the logging is throughout secure messaging but the structlog format we're using now would have this log look like
`log.info('Sent secure message email notification sent via RM Notify-Gateway to GOV.UK Notify', message_id=response.json()["id"])